guides/common/modules/proc_using-project-as-a-source-of-content-for-imagebuilder.adoc

```diff
@@ -0,0 +1,21 @@
+[id="using-project-as-a-source-of-content-for-imagebuilder"]
+= Using {Project} as a source of content for {LoraxCompose}
+
+In {Project}, you can integrate with {Cockpit}.
+By using {Cockpit}, you can access {LoraxCompose} and build images.
+When you configure {LoraxCompose} for image building, you can use {Project} content to build the images.
+
+.Prerequisites
+* Ensure you have imported content into {ProjectServer}.
```

Comment:
Not sure that you require this prerequisite. Would a user try to build an image if there is no content?